Sources/LoopUITestingKit/Screens/BaseScreen.swift

```diff
@@ -7,6 +7,10 @@
 
 import XCTest
 
+extension XCUIElementQuery {
+    var lastMatch : XCUIElement {return self.element(boundBy: self.count - 1)}
```

Comment:
Please fix code formatting to use one space after `{` and before `}`
```suggestion
    var lastMatch : XCUIElement { return self.element(boundBy: self.count - 1) }
```

Reply:
Formatting fixed